.travis_scripts/openapi-validator.sh

```diff
@@ -1,7 +1,25 @@
 #!/usr/bin/env bash
-set -e
+set -eu
+set -o pipefail
+
+function download_generator {
+  curl -L https://raw.githubusercontent.com/OpenAPITools/openapi-generator/master/bin/utils/openapi-generator-cli.sh -o ./openapi-generator-cli
+  chmod +x ./openapi-generator-cli
```

Comment:
@bzwei Why not have the retry logic here
you could check $? to get the exit code of the curl command and then retry the download again.
I am not sure if we have to to run the validate in order to figure out that the download failed and then issue a retry.

Reply:
The download normally has no problem. Running the command line may have the exit code nonzero either because the true validation fails (desired) or the maven build fails (undesired, retry).

Reply:
The script first build a jar file and then run the jar to validate openapi.